zebrad/src/components/tracing/component.rs

```diff
@@ -77,7 +77,7 @@ impl<A: abscissa_core::Application> Component<A> for Tracing {
     }
 
     fn version(&self) -> abscissa_core::Version {
-        abscissa_core::Version::parse("1.0.0-alpha.6").unwrap()
+        abscissa_core::Version::parse(env!("CARGO_PKG_VERSION")).unwrap()
```

Comment:
This change makes our release process a bit simpler.

So we can delete the "zebrad app code" from our release checklist:
https://github.com/ZcashFoundation/zebra/blob/main/.github/PULL_REQUEST_TEMPLATE/release-checklist.md#version-locations

Reply:
I added this change to the PR branch.